backend/backend.proto

```diff
@@ -26,6 +26,17 @@ service Backend {
   rpc StoresFind(StoresFindOptions) returns (StoresFindResult) {}
 
   rpc Rerank(RerankRequest) returns (RerankResult) {}
+
+  rpc GetMetrics(MetricsRequest) returns (MetricsResponse);
+}
+
+// Define the empty request
+message MetricsRequest {}
+
+message MetricsResponse {
```

Comment:
what other metrics are good to be exposed out?

Reply:
Maybe not related to the processing request, but could be useful also to expose infos like how many parallel requests can be served